spec/Gemfile.beaker

```diff
@@ -0,0 +1,8 @@
+source "https://rubygems.org"
+
+group :system_tests do
+  gem 'beaker', :github => 'petems/beaker', :branch => 'MAINT-fix_mkdir_bug', :require => false
```

Comment:
Yeah, this is just because there was a bug on master, but I think its fixed by now, I'll change this back